repo/linux/velinux

```diff
@@ -0,0 +1,28 @@
+url: https://github.com/bytedance/kernel.git
```

Comment:
thanks, help name the repo as bytedance, per https://github.com/intel/lkp-tests/wiki/Repo-Naming-Convention